apps/backend/src/modules/ai-assistant/ai-assistant.controller.ts

```diff
@@ -486,11 +618,11 @@ class AIAssistantController extends BaseController {
 	 *           schema:
 	 *             type: object
 	 *             properties:
-	 *               categories:
+	 *               payload:
```

Comment:
Let's have `categories` instead of `payload`, it will be more clear

Reply:
I use the payload to be able to send different types in request:
`type AIAssistantRequestDto = {
	payload: SelectedCategory[] | TaskCreateDto | TextMessage;
	threadId: string;
};`

Do you mean to change the swagger schema + dto + validation?

Reply:
We need to have a separate type for each endpoint, not combine everything